apps/commudle-admin/src/app/app.module.ts

```diff
@@ -119,24 +119,10 @@ import { SkeletonScreensModule } from './feature-modules/skeleton-screens/skelet
 import { UserChatsModule } from './feature-modules/user-chats/user-chats.module';
 import { UsersModule } from './feature-modules/users/users.module';
 import { AppInitService } from './services/app-init.service';
-import { CommunitiesCardComponent } from 'apps/commudle-admin/src/app/app-shared-components/communities-card/communities-card.component';
-import { LoginConsentPopupComponent } from './components/login-consent-popup/login-consent-popup.component';
```

Comment:
@arpansac @ArshdeepGrover fyi, fixed this import de-duplication as part of https://github.com/commudle/commudle-ng/pull/1500